.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,6 +6,7 @@
 - [ ] (If applicable) I've documented below the tests I've run on the DLC image
 - [ ] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
 - [ ] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.
+- [ ] (If applicable) I've added the markers `@pytest.mark.integration("<feature-being-tested>")` and `@pytest.mark.model("<model-type>")` to the new tests which I have added
```

Comment:
model is the only required marker I believe. if you are going to add integration, can you also add `and @pytest.mark.multinode(num_nodes) if the test runs on more than one instance`

Reply:
Added. Please take a look at the updated template.